opsdroid/events.py

```diff
@@ -284,6 +284,9 @@ async def respond(self, response_event):
 
         return await super().respond(response)
 
+    async def remove(self, reason=None):
+        return await super().respond(DeleteMessage(reason, linked_event=self))
+
```

Comment:
This is the only bit of this PR that I am unsure on. I don't know if we want to add this helper method or not.

We don't have a helper method for `react` or any other specific type of response. I shall ask the other maintainers what their thoughts are.